router/xgress_edge_tunnel/fabric.go

```diff
@@ -48,6 +49,10 @@ import (
 	"time"
 )
 
```

Comment:
I feel like we should add this to foundation. Tiny, but generally useful thing

Reply:
Looks like it is already there in foundation/util/util.go. It is called `Ptr()`